_docs/backenders/strategy-2021/roles/hiring.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Template
+parent: Roles
+grand_parent: Strategy 2021
+---
+
+# Capacity role definition
+
+**Current responsible**: -
+
+
+
+***Note:***
+
+2 people Identify the hiring related tasks to be managed between different roles:
```

Comment:
Perhaps all these roles should be involved in the advice process on hiring. Might need adding to the "decision making" for these roles' descriptions. https://github.com/Vizzuality/devismos/pull/14 https://github.com/Vizzuality/devismos/pull/15 https://github.com/Vizzuality/devismos/pull/16